lms/templates/problem.html

```diff
@@ -55,6 +55,10 @@ <h2 class="problem-header">
     % endif
     % if check_button:
     <input class="check ${ check_button }" type="button" data-checking="${ check_button_checking }" value="${ check_button }" />
+    %elif exceeded_time_limit:
+    <p class="exceeded-time-limit-message">
+      Deadline has passed, you can no longer submit.
```

Comment:
This is joining independent clauses; should this be a semicolon instead of a comma?